GetUserServices/handler.ts

```diff
@@ -0,0 +1,142 @@
+import * as express from "express";
+
+import {
+  ClientIp,
+  ClientIpMiddleware
+} from "io-functions-commons/dist/src/utils/middlewares/client_ip_middleware";
+
+import {
+  AzureApiAuthMiddleware,
+  IAzureApiAuthorization,
+  UserGroup
+} from "io-functions-commons/dist/src/utils/middlewares/azure_api_auth";
+import {
+  AzureUserAttributesMiddleware,
+  IAzureUserAttributes
+} from "io-functions-commons/dist/src/utils/middlewares/azure_user_attributes";
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "io-functions-commons/dist/src/utils/request_middleware";
+import {
+  IResponseSuccessJson,
+  ResponseSuccessJson
+} from "italia-ts-commons/lib/responses";
+import { EmailString, NonEmptyString } from "italia-ts-commons/lib/strings";
+
+import {
+  checkSourceIpForHandler,
+  clientIPAndCidrTuple as ipTuple
+} from "io-functions-commons/dist/src/utils/source_ip_check";
+
+import { Context } from "@azure/functions";
+import { identity } from "fp-ts/lib/function";
+import {
+  fromLeft,
+  taskEither,
+  TaskEither,
+  tryCatch
+} from "fp-ts/lib/TaskEither";
+import { ServiceId } from "io-functions-commons/dist/generated/definitions/ServiceId";
+import { ServiceModel } from "io-functions-commons/dist/src/models/service";
+import { ContextMiddleware } from "io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { UserInfo } from "../generated/api-admin/UserInfo";
+import { ServiceIdCollection } from "../generated/definitions/ServiceIdCollection";
+import { APIClient } from "../utils/clients/admin";
+import { getLogger, ILogger } from "../utils/logging";
+import {
+  ErrorResponses,
+  toDefaultResponseErrorInternal,
+  toErrorServerResponse
+} from "../utils/responses";
+
+/**
+ * Type of a GetUserServices handler.
+ *
+ * GetUserServices returns a list of ServiceId as output.
+ */
+type IGetUserServicesHandler = (
+  context: Context,
+  auth: IAzureApiAuthorization,
+  clientIp: ClientIp,
+  attrs: IAzureUserAttributes
+) => Promise<IResponseSuccessJson<ServiceIdCollection> | ErrorResponses>;
+
+const logPrefix = "GetUserServicesHandler";
+
+const getUserServicesTask = (
```

Comment:
This method returns the user and not only the services.
You need to change the name to getUserTask or modify it so it returns only the user services.

Reply:
Fixed in [2f63ab6](https://github.com/pagopa/io-functions-services/pull/68/commits/2f63ab68bf93f2db5b11b603e52fa36b04de3306)